rest/data/commit_queue.go

```diff
@@ -373,10 +373,19 @@ func getPRAndCheckMergeable(ctx context.Context, env evergreen.Environment, sc C
 	if err != nil {
 		return nil, err
 	}
-
+	mergeableState := pr.GetMergeableState()
+
+	grip.Info(message.WrapError(err, message.Fields{
```

Comment:
If you wrap error here, you'll only actually log if the error isn't nil -- not sure if that's what you want.

Reply:
Can we also use grip.Debug since this is for debugging?